sdks/python/apache_beam/ml/inference/huggingface_inference.py

```diff
@@ -593,9 +594,11 @@ def __init__(
     Args:
       task (str or enum.Enum): task supported by HuggingFace Pipelines.
         Accepts a string task or an enum.Enum from PipelineTask.
-      model : path to pretrained model on Hugging Face Models Hub to use custom
-        model for the chosen task. If the model already defines the task then
-        no need to specify the task parameter.
+      model : path to the pretrained model-id on Hugging Face Models Hub
```

Comment:
is this some full path or just the model id itself?

Reply:
also, please add a type `(str):`, i am guessing.

Reply:
just the model id